main.go

```diff
@@ -107,6 +107,7 @@ func main() {
 
 		if build.Status != 0 {
 			if strings.TrimSpace(cfg.BuildArtifactsSavePath) != "" {
```

Comment:
NOTE: `strings.TrimSpace(cfg.BuildArtifactsSavePath)` could be assigned to a new variable, for instance:

```
buildArtifactSaveDir := strings.TrimSpace(cfg.BuildArtifactsSavePath)
```

this way you do not need to duplicate `strings.TrimSpace(fullBuildArtifactsSavePath)`.